tests/tpch/conftest.py

```diff
@@ -30,9 +30,6 @@ def pytest_addoption(parser):
     parser.addoption("--cloud", action="store_false", dest="local", help="")
     parser.addoption("--restart", action="store_true", default=True, help="")
     parser.addoption("--no-restart", action="store_false", dest="restart", help="")
-    parser.addoption(
-        "--performance-report", action="store_true", default=False, help=""
-    )
     parser.addoption(
```

Comment:
Not clear to me why this is being removed, but that's okay. Just wanted to make sure this is intentional is all

Reply:
This PR included https://github.com/coiled/benchmarks/pull/1599.